src/main/java/com/epam/processor/DataProcessor.java

```diff
@@ -60,7 +86,35 @@ public RoadAccident getAccidentByIndex7(String index){
      * @return
      */
     public List<String> getTopThreeWeatherCondition7(){
-        return null;
+    	Map<String, Integer> map = new HashMap<String, Integer>();
```

Comment:
too long method - each for should be a separate method.  On next HW this will decrease mark. 
If you see empty lines as delimiters - think , maybe need a new method.   
If method more that 5 strings long, try to extract.